README.md

```diff
@@ -1,92 +1,50 @@
+3. `cdf_deployment_credentials`: The API-key that will be used to deploy the function. It must have the following CDF capabilities: `Files:READ`, `Files:WRITE`, `Functions:READ`, `Functions:WRITE`. You can scope the files-access to a dataset (see 'data_set_external_id')`).
+4. `cdf_runtime_credentials`: The API-key that the function will use when running "inside" of Cognite Functions. It must have the CDF capabilities required to run your code.
+Example: if your code has to read assets, and write to timeseries, it will need `Assets:READ` and `TimeSeries:Write`.
 
 #### Optional
-1. `common_folder`:  The path to the folder containing code that is shared between all functions. Defaults to `common/`
-1. `cdf_project`: The name of your CDF project/tenant. Inferred from your API-keys. Will be validated with API-keys if provided
-2. `cdf_base_url`: Base url of your CDF tenant, defaults to _https://api.cognitedata.com_
+1. `common_folder`:  The path to the folder containing code that is shared between all functions. Defaults to `common/`. More information below.
+1. `cdf_project`: The name of your CDF project/tenant. Will be inferred from your API-keys and used to validate against, if given.
+2. `cdf_base_url`: Base url of your CDF tenant, defaults to `https://api.cognitedata.com`.
 3. `function_file`: The name of the file with your main function (defaults to `handler.py`)
-4. `function_secrets`: The name of Github secrets that holds the base64 encoded JSON dictionary with secrets. (see secrets section)
-5. `schedule_file`: File location with the list of schedules to be applied, see the file format below (defaults to None i.e. no schedules).
-6. `remove_only`: Checks that specified function is removed with all its schedules. Deployment logic is skipped
-7. `data_set_external_id`: [recommended] Data set external ID to use for the function-associated file (zipped code folder).
-8. `cpu`: Set number of CPU cores per function. See defaults and allowed values in the [api documentation](https://docs.cognite.com/api/playground/#operation/post-api-playground-projects-project-functions).
-9. `memory`: Set memory per function measured in GB. See defaults and allowed values in the [api documentation](https://docs.cognite.com/api/playground/#operation/post-api-playground-projects-project-functions).
-10. `owner`: Used to specify a function's owner. See allowed number of characters in the [api documentation](https://docs.cognite.com/api/playground/#operation/post-api-playground-projects-project-functions)
+4. `function_secrets`: The name of a Github secret that holds the base64 encoded JSON dictionary with secrets. (see secrets section)
+5. `schedule_file`: File location inside `function_folder` containing a list of schedules to be attached to your function. Check out the details in the section below. Note: Ignored with warning if pointing to a non-existent file.
+6. `remove_only`: Deletes function along with all attached schedules. Deployment logic is skipped.
```

Comment:
We don't have a use for `remove_only` anymore, since "cleanup PR yaml" is going away. I still think we should keep it as an option in here, so that a short-cut for "delete/cleanup" is available. Opinions @pavelzubarev @ks93 ? 🤔

Reply:
Then I think my suggestion is an even better alternative 😅